Formula/m/mariadb@10.9.rb

```diff
@@ -1,10 +1,9 @@
 class MariadbAT109 < Formula
   desc "Drop-in replacement for MySQL"
   homepage "https://mariadb.org/"
-  url "https://downloads.mariadb.com/MariaDB/mariadb-10.9.7/source/mariadb-10.9.7.tar.gz"
-  sha256 "59c4b178eb859ead207d5c0a7515263c734dc0622af46e1657ef12cb81bc10f5"
-  license "GPL-2.0-only"
-  revision 1
+  url "https://archive.mariadb.org/mariadb-10.9.8/source/mariadb-10.9.8.tar.gz"
+  sha256 "fcdc6f73882477c20310e795a02f746e54934aa7a8b368900bd45e78a5daf6ed"
+  license "GPL-2.0-or-later"
```

Comment:
Corrected the license; see https://github.com/MariaDB/server/blob/mariadb-10.9.8/COPYING#L299